packages/@n8n/task-runner/src/config/base-runner-config.ts

```diff
@@ -14,8 +14,8 @@ class HealthcheckServerConfig {
 
 @Config
 export class BaseRunnerConfig {
-	@Env('N8N_RUNNERS_N8N_URI')
-	n8nUri: string = '127.0.0.1:5679';
+	@Env('N8N_RUNNERS_TASK_BROKER_URI')
+	taskBrokerUri: string = '127.0.0.1:5679';
```

Comment:
Shouldn't this have scheme?

https://github.com/n8n-io/task-runner-launcher/blob/37515673b39f9b61e996a98002543b0c638c019f/internal/config/validate_url_test.go#L29-L35